DependencyInjection/CmfCoreExtension.php

```diff
@@ -118,9 +118,6 @@ public function prepend(ContainerBuilder $container)
                         break;
                     case 'cmf_routing':
                         $routePaths = array($persistenceConfig['basepath'].'/routes');
```

Comment:
guess its a topic for https://github.com/symfony-cmf/RoutingBundle/pull/311 - if at all.